packages/turbo/android/src/main/java/com/reactnativeturbowebview/RNVisitableViewManager.kt

```diff
@@ -70,7 +71,8 @@ class RNVisitableViewManager(
           root.injectJavaScript(it)
         }
       }
-      RNVisitableViewCommand.RELOAD_PAGE -> root.refresh(true)
+      RNVisitableViewCommand.RELOAD_PAGE -> root.reload()
+      RNVisitableViewCommand.REFRESH_PAGE -> root.refresh(true)
```

Comment:
Same as iOS here. We do not want to show progress when _refreshing_ only when _reloading_.